### PR TITLE
Collections hw

### PR DESCRIPTION
**Pull request by hrytsyshynq**

### CODE REVIEW

File: src/main/java/library/Template.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import library.enumeration.BookAuthor;
+import library.enumeration.BookGenre;
+import library.enumeration.BookLanguage;
+
+import java.util.*;
+
+public class Template {
+
+    private List<BookItem> booksList;
+    List<BookItem> resultList;
```
Comment (olobanok) [inline src/main/java/library/Template.java]: it is better to specify access modifier explicitly rather then use default all the time

File: src/main/java/library/common/Common.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package library.common;
+
+import library.BookItem;
+
+import java.util.List;
+import java.util.Random;
+
+public class Common {
```
Comment (olobanok) [inline src/main/java/library/common/Common.java:8]: if you create utility class where all methods are static, it is a good practice to add private constructor

File: src/main/java/library/common/Common.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    }
+
+    public static String setCommaAsDelimiter(List<BookItem> itemList) {
+        StringBuffer sb = new StringBuffer();
+        for (BookItem item : itemList){
+            sb.append(item.getTitle()).append(", ");
+        }
+        sb.deleteCharAt(sb.lastIndexOf(","));
+        return sb.toString();
+    }
```
Comment (olobanok) [inline src/main/java/library/common/Common.java:29]: you can also replace this method with String.join() that can write a collection to string with a specified delimiter

**Review by olobanok — APPROVED** [review]

File: src/main/java/library/Template.java (hunk 1 above)
Comment (hrytsyshynq) [reply src/main/java/library/Template.java]: fixed

File: src/main/java/library/common/Common.java (hunk 2 above)
Comment (hrytsyshynq) [reply src/main/java/library/common/Common.java:8]: fixed

**Pull request merged**